File: refactored/ignored-test/CSharpXUnit/CSharpXUnit.Tests/Tests/ItCallsSearchServiceOnce.cs
using Xunit;
using Moq;

public class Search {
    // [Fact(Skip = "// @Disabled")]
    [Fact]
    public void itCallsSearchServiceOnce() {
        // Arrange
        // Act
        _controller.Search("jos");
        // Assert
        _speakerServiceMock.Verify(m => m.Search(It.IsAny<string>()), Times.Once);
    }
}

File: refactored/ignored-test/java-junit/ItCallsSearchServiceOnce.java
import org.junit.Test;
import org.mockito.Mockito;

public class Search {
    @Test
    // @Disabled
    public void itCallsSearchServiceOnce() {
        // Arrange
        // Act
        _controller.search("jos");
        // Assert
        Mockito.verify(_speakerServiceMock, Mockito.times(1)).search(Mockito.anyString());
    }
}
